### PR TITLE
screens

### PR DESCRIPTION
**Pull request by jhonwms**

### CODE REVIEW

File: src/app/components/common/ItemList/index.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+const ListElements = ({
+  title, values, callback, withFilter,
+}) => {
+  const [listItems, filterList] = useState(values);
+  const [activeElement, setActive] = useState({});
+
+  const memoCallback = useCallback((text) => filterList(!text
+    ? values : listItems
+      .filter((field) => field.name.toLowerCase().includes(text.toLowerCase()))),
```
Comment (usaqui-albert) [inline src/app/components/common/ItemList/index.js:44]: Try splitting this block of code into pieces to make it more readable!

File: src/app/components/common/ItemList/index.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      />
+    </View>
+  );
+};
+
+export default ({
+  title, closeModalCallBack, values, selectedItemCallBack, visible, withFilter,
+}) => (
+  <Modal
+    style={{
```
Comment (usaqui-albert) [inline src/app/components/common/ItemList/index.js:84]: Did you try styling this component using `styled-components`?

File: src/app/components/common/ItemList/index.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+      justifyContent: 'flex-end',
+    }}
+    modalAnimation={slideAnimation}
+    coverScreen
+    modalTitle={(
+      <HeaderList
+        title={title}
+        closeModal={() => { closeModalCallBack(); }}
```
Comment (usaqui-albert) [inline src/app/components/common/ItemList/index.js:92]: Why do not just use `closeModal={closeModalCallBack}`

File: src/app/components/common/ItemList/style.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
+// @flow
+import styled from 'styled-components';
+import { theme } from 'app/themes';
+
+const defaultRoundedColor = '#673AB7';
```
Comment (usaqui-albert) [inline src/app/components/common/ItemList/style.js:5]: Please add this color in Themes

File: src/app/components/common/ItemList/style.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  flex-direction: row;
+  border-bottom-width: 1;
+  border-color: ${theme.colors.lightGray};
+  padding-bottom: 15;
+`;
+
+export const LabelHeader = styled.Text`
+  width: 30;
+  height 30;
+  color: #323275;
```
Comment (usaqui-albert) [inline src/app/components/common/ItemList/style.js:21]: Also this one

File: src/app/components/common/ItemList/style.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  font-size: 14;
+`;
+
+export const FilterInput = styled.TextInput`
+  margin-bottom: 10;
+  font-family: ${theme.fonts.SfProMedium};
+  border-bottom-width: 20;
+  margin-horizontal: 55;
+  font-size: 16;
+  borderColor: rgba(255,255,255, .21);
```
Comment (usaqui-albert) [inline src/app/components/common/ItemList/style.js:76]: I see another usage of this white-transparent color. Wouldn't be that bad creating a `white21` in Themes.

File: src/app/screens/MailingAddress/index.js [hunk 7]
```diff
@@ -68,17 +67,19 @@ const MailingAddress = ({ holdsInfoForm, fields }) => {
           <HeaderText>Mailing Address</HeaderText>
         </ContentHeaderInfo>
         <ContainerSeparator>
-          {formsInputs.map((item, index) => (item.type === 'list'
+          {fields.map((item, index) => (item.type === 'list'
             ? (
-              <CountriesList />
+              <CountriesList setInParent={({ id }) => {
+                console.log(fields);
```
Comment (usaqui-albert) [inline src/app/screens/MailingAddress/index.js:73]: Get rid of `console.log`'s

File: src/app/components/common/ItemList/index.js (hunk 2 above)
Comment (jhonwms) [reply src/app/components/common/ItemList/index.js:106]: yes but has an error with the childcomponent :/

File: src/app/components/common/ItemList/index.js (hunk 3 above)
Comment (jhonwms) [reply src/app/components/common/ItemList/index.js:114]: likeit

**Pull request merged**